.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,34 @@
+name: Docker image build
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  docker-build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        lambda:
+          - blazer
+          - google-cidr
```

Comment:
So one docker to rule them all? (i.e. on docker image for all lambdas)

Reply:
Nope, each Lambda will have its own Dockerfile in its folder.  That'll give us the flexibility to tailor them to the specific Lambda's needs and language.